packages/arb-token-bridge-ui/src/components/common/HeaderAccountPopover.tsx

```diff
@@ -154,17 +154,17 @@ export function HeaderAccountPopover({
           src={ensAvatar || undefined}
           className="h-6 w-6 rounded-full sm:h-8 sm:w-8"
           fallback={
-            <CustomBoringAvatar size={isSmallScreen ? 30 : 40} name={address} />
+            <CustomBoringAvatar size={isSmallScreen ? 24 : 40} name={address} />
           }
         />
-        <span className="text-lg text-white sm:text-base">
+        <span className="text-base text-white/60 sm:text-white">
           {ensName ?? udInfo.name ?? accountShort}
         </span>
 
-        <ChevronDownIcon className="ml-auto h-3 w-3 text-white" />
+        <ChevronDownIcon className="ml-auto h-[16px] w-[16px] text-white/60 transition duration-200 sm:text-white" />
```

Comment:
Why are we using 4px base in the header? Is it because of the sidebar so we're compatible with the Portal?

Otherwise we should round down to `h-3 w-3`